lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -22,32 +22,35 @@ define('DOSOMETHING_CAMPAIGN_PIC_STEP_HEADER', t('Snap a Pic'));
 function dosomething_campaign_form_campaign_node_form_alter(&$form, &$form_state, $form_id) {
   unset($form['field_active_hours'][LANGUAGE_NONE]['#options']['_none']);
   $form['title']['#description'] = t('Title - i.e. "Protect your Pills", "Birthday Mail", or "Teens for Jeans" <br/><strong> Limit: 20 characters. </strong>');
-  // Add in campaign close info.
-  dosomething_campaign_run_add_campaign_run_info($form, $form_state);
   // Use #after_build to add JS even on validation errors: https://drupal.org/node/1253520#comment-4881588
   $form['#after_build'][] = 'dosomething_campaign_form_campaign_node_form_after_build';
 
   $nid = $form['nid']['#value'];
+  $language = $form['field_current_run']['#language'];
+  $all_options = $form['field_current_run'][$language]['#options'];
 
   // If this is an /add/ form then nid won't be set yet.
   if (!is_null($nid)) {
-    $campaign_runs = dosomething_campaign_get_runs($nid);
+    $campaign_runs = dosomething_campaign_get_runs($nid, $language);
   }
 
   // Only show the campaign runs tied to the campaign as options.
-  if (isset($campaign_runs) && !empty($campaign_runs)) {
-    $language = $form['field_current_run']['#language'];
-    $all_options = $form['field_current_run'][$language]['#options'];
-
-    foreach ($campaign_runs as $run) {
-      $filtered_options[$run->entity_id] = $all_options[$run->entity_id];
-    }
-
+  if (isset($campaign_runs)) {
+    $options = $campaign_runs;
     // Add empty option to the options array. Helps to ensure that the user
     // explicity chooses a campaign instead of defaulting to the first option
     // in the list.
-    $filtered_options = array("_none" => "- None -") + $filtered_options;
-    $form['field_current_run'][$language]['#options'] = $filtered_options;
+    $options = array("_none" => "- None -");
+
+    // If there are campaign runs tied to this campaign translation.
+    if (!empty($campaign_runs)) {
+      foreach ($campaign_runs as $run) {
+        $options[$run->entity_id] = $all_options[$run->entity_id];
+      }
+    }
+
```

Comment:
@angaither Now the helper text that used to show the current run under status field will show under the current run drop down. If there is no current run selected, the text will instruct them to select a run.